jiant/utils/config.py

```diff
@@ -1,17 +1,18 @@
-from typing import Type, Union, Sequence, Iterable
-from . import hocon_writer
-import json
-import pyhocon
-import argparse
 '''Train a multi-task model using AllenNLP '''
+from typing import Type, Union, Sequence, Iterable
 import os
 import sys
 import time
-import random
+import json
 import types
+import random
+import pyhocon
+import argparse
 import logging as log
+
 log.basicConfig(format='%(asctime)s: %(message)s',
                 datefmt='%m/%d %I:%M:%S %p', level=log.INFO)
+from . import hocon_writer
```

Comment:
should this be `from jiant.utils import hocon_writer` ?

Reply:
I didn't change relative imports that were really deep in the repo and importing from the same directory. You think it's better to switch everything to absolute imports?

Reply:
Yes, for consistency and less brittle if we move anything.